tasks/src/main/java/ru/sberbank/school/task10/ThreadPool.java

```diff
@@ -0,0 +1,28 @@
+package ru.sberbank.school.task10;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.Future;
+
+public interface ThreadPool {
+    /**
+     * Запускает потоки, которыые бездействуют, до тех пор пока не появится новое задание в очереди (см. execute)
+     */
+    void start();
+
+    /**
+     * Отменяет запланированные задачи, останавливает запущенные задачи, останавливает потоки, переходит в начальное состояние.
```

Comment:
Line is longer than 120 characters (found 127).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-797535044>*